packages/reffects/index.d.ts

```diff
@@ -0,0 +1,32 @@
+interface Event {
+    id: string;
+    payload?: object;
+}
+
+interface Effect {
+    id: string;
+    data: object;
+}
+
+interface Coeffect{
+    id: string;
+    data?: object;
+}
+
+type EventHandler = (coeffect?: Coeffect, payload?: object) => Effect;
```

Comment:
This should return an object with many Effects inside, not sure how to represent it in TypeScript :)

Reply:
Just to clarify, this is how looks a event handler return:

```
return {
        'state.set': {
          todos: newTodos,
        },
        toast: {
          text: `"${text}" was marked as ${isDone ? 'undone' : 'done'}.`,
          milliseconds: 3000,
        },
      };
```